ext/libyuv.cmd

```diff
@@ -6,6 +6,9 @@
 
 : # If you're running this on Windows, be sure you've already run this (from your VC2019 install dir):
 : #     "C:\Program Files (x86)\Microsoft Visual Studio\2019\Professional\VC\Auxiliary\Build\vcvars64.bat"
+: # It is recommended to build libyuv with clang-cl on Windows, because most libyuv's assembly code is written in GCC
+: # inline assembly syntax which MSVC doesn't support. Run this if you have clang-cl installed:
```

Comment:
Nit: add "of" after "most".

Add a comma `,` between "syntax" and "which".

Keep the line length under 100 characters.

```
: # It is recommended to build libyuv with clang-cl on Windows, because most of libyuv's assembly
: # code is written in GCC inline assembly syntax, which MSVC doesn't support. Run this if you have
: # clang-cl installed:
```